ystia/yorc-a4c-plugin
# Feature/brbdcf 1302 gcp compute instance deployment

# Pull Request description

## Description of the change

Associated to Yorc Pulll Request https://github.com/ystia/yorc/pull/33

### What I did

Added a yorc.nodes.google.Compute type and a Google Cloud location.

Added a TOSCA sample described at tosca-samples/org/ystia/yorc/samples/vision/README.md

Updated the documentation at documentation/location.rst

### How to verify it

See associated Yorc pull request to configure Yorc with a Google Cloud Infrastructure.
Then see this pull request documentation/location.rst  to configure a Google Cloud location.
A Tosca sample is also provided in this Pull Request at tosca-samples/org/ystia/yorc/samples/vision
Follow the intsructions in this sample README to deploy the sample and check the application works
